### ISSUE TITLE
Subnetworks list doesn't update after creation

### ISSUE THREAD

**Issue by danyill** [openscd/open-scd#810]
**Describe the bug**

After creating a subnetwork it does not appear in the list.

**To Reproduce**

1. Open the file: test/testfiles/zeroline/functions.scd
2. Click on the Communication Editor.
3. Click Add Subnetwork.
4. Add some parameters.
5. Click "Add +"
6. The list of subnetworks is unchanged.
7. _However_ if the user goes to a different editor and then returns to the Communication editor the new subnetwork is shown.

**Expected behavior:**

After clicking "Add +" the subnetwork list should be refreshed.
I suspect this may be a consequence of a recent refactor.

**Desktop:**

- OS: Linux, Kubuntu 22.04
- Browser: Chrome
- Version: Version 102.0.5005.61 (Official Build) (64-bit)

**Additional context:**

None

**Comment by dlabordus** [comment]
The same problem is in the IED Editor (and our new 104 Plugin we are buildin)g.
In the IED Editor when you update the name of the IED, the change doesn't show directly, only after switching editors and back again. 
Also when you initialize a DA Value in the IED Editor, that also not directly reflected. Only after collapsing the LN and expanding it again.

**Comment by dlabordus** [comment]
There is also some strange behavior in the Template Editors. All 4 type lists have the same problem.
When you edit 1 type and change the name it is displayed directly, but you aren't able to edit the same type directly again.
If you first edit a other type and then try editing the first one again it works again.

**Comment by ca-d** [comment]
> The same problem is in the IED Editor

Too bad we didn't catch this in review, the fix is already merged as is, so we'll need a second pull request. The original issue (Subnetworks list doesn't update) has been fixed by it, at least. I just confirmed this.

The remaining problem with the IED seems to be (at first glance) that you have a separate `selectedIed` property there in which you set an `Element` manually (instead of reactively) and that isn't updated when `doc` is. It looks like you did this because you needed to remember the currently selected IED in a global variable before the change to how we load plugins. This should no longer be the case.

You can see the symptoms of this by looking at the `alphabeticOrderedIeds` listed in the dropdown at the top: The list items themselves are updated as soon as you update the IED (new name immediately reflected), while anything rendered based on `this.selectedIed` stays the same. You only actually `requestUpdate('selectedIed')` `onSelect`, where this *should* be unnecessary as long as the `selectedIed` reference has actually changed.

I suggest adding a `@query` to the current `mwc-list-item["selected"]` and using `this.selectedIEDListItem` in a reactive `private get selectedIED(): string` with no setter, turning `selectedIED` into a fully reactive property, and passing the data down in a way similar to what we do in the Substation editor.

I also suggest closing this issue and creating new issues for any other problems we find with individual plugins' compatibility with the new updating scheme. What do you think?

**Comment by ca-d** [comment]
Duplicate of https://github.com/openscd/open-scd/issues/813, fixed by https://github.com/openscd/open-scd/pull/814 .
